lib/Core/Executor.cpp

```diff
@@ -127,10 +127,26 @@ namespace {
 
 /*** Test generation options ***/
 
-cl::opt<bool> DumpStatesOnHalt(
-    "dump-states-on-halt",
-    cl::init(true),
-    cl::desc("Dump test cases for all active states on exit (default=true)"),
+// HACK: we can't specify an init value and therefore use "none" as special value
+// to indicate an empty list; an empty list means _all_ tests are active (default)
+cl::bits<StateTerminationClass> WriteTests(
+    "write-tests",
+    cl::desc("Select termination classes for which tests should be generated "
+             "(default=exit,early,error,user-error,execution-error)"),
+    cl::values(
+        clEnumValN(StateTerminationClass::NONE, "none", "Disable test generation"),
```

Comment:
The thing is that you can't specify the default value here. That means an "empty" option has to be treated as default (write tests) instead of writing no tests. NONE is used to signal that you actually don't want any tests.

Reply:
The phrasing is a bit confusing to me. Maybe change `none` to `no-tests` and the comment should read "... there we use 
 no-tests to disable test generation and an empty list means __all___ test are active"